src/swap.cpp

```diff
@@ -0,0 +1,19 @@
+/*
+OVERVIEW: Write a function which takes two numbers as input and swaps those two numbers.
+E.g.: int a = 1, b = 2; after swap(a, b) call value of "a" should be 2 and value of "b" should be 1.
+
+INPUTS:  Two integer pointers a, b.
+
+OUTPUT:  Swap the values of a, b.
+
+ERROR CASES:
+
+NOTES:
+*/
+
+void swap(int *a, int *b)
+{
+	*a = *a + *b;
+	*b = *a - *b;
+	*a = *a - *b;
+}
```

Comment:
The solution is okay. But, the addition and subtraction logic won't work good for pointers and references in all programming languages. So my suggestion is to be aware of what you're dealing with. In C it is okay, but that doesn't hold the same for all other languages. Add and Sub are suggested when you're dealing with values. Even I'm not sure that if this solution can cause problems in other langs but this is a bit of speculation and suggestion from my side.

Reply:
yeah okay Anna.